openedx/features/job_board/constants.py

```diff
@@ -1,16 +1,35 @@
+JOB_PARAM_QUERY_KEY = 'query'
+JOB_PARAM_COUNTRY_KEY = 'country'
+JOB_PARAM_CITY_KEY = 'city'
+
+JOB_TYPE_REMOTE_INDEX = 0
+JOB_TYPE_ONSITE_INDEX = 1
```

Comment:
Good effort, but instead of defining CONSTANTS for indexes, how about if we define constants for those values we need to use again and again. For example
```
JOB_TYPE_REMOTE = 'remote'
JOB_TYPE_ONSITE = 'onsite'

JOB_TYPE_CHOICES = (
      (JOB_TYPE_REMOTE, 'Remote'),
      (JOB_TYPE_ONSITE, 'Onsite'),
)
```

Then point where we are fetching param

```
remote = params.get(JOB_TYPE_REMOTE)
onsite = params.get(JOB_TYPE_ONSITE)
```

and the point where we are combining them

```
job_type_list = [job_type for job_type in (JOB_TYPE_REMOTE if remote else '', JOB_TYPE_ONSITE if onsite else '') if job_type]
```

Reply:
or even if we follow the the comment in theme to put actual values then

```
job_type_list = [job_type for job_type in (remote , onsite ) if job_type]
```